api/CameraSetPosCommand.md

```diff
@@ -8,15 +8,15 @@
 
 `camera_set_pos speed x y`
 
-Moves the camera to the given position.
+Moves the camera to the given absolute position over a time period.
 
 **Parameters**
 
 - *speed*: Number of seconds the transition should take
 - *x*: Target X coordinate
 - "y*: Target Y coordinate
 
-For more details see: https://docs.escoria-framework.org/camera
+For more details see: https://docs.escoria-framework.org/en/devel/getting_started/camera.html
```

Comment:
@dploeger same question as above, and it will be the same question for any other similarly-updated links in this PR.